moninag/package.json

```diff
@@ -1,63 +1,64 @@
 {
-    "name": "moninag",
-    "version": "1.0.0",
-    "description": "Monitoring system",
-    "main": "source_app.js",
-    "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/Lv-219-Python/MoniNag.git"
-    },
-    "author": "Lv-219.Python",
-    "license": "ISC",
-    "bugs": {
-        "url": "https://github.com/Lv-219-Python/MoniNag/issues"
-    },
-    "homepage": "https://github.com/Lv-219-Python/MoniNag#readme",
-    "devDependencies": {
-        "@types/core-js": "^0.9.35",
-        "@types/node": "^7.0.5",
-        "angular2-template-loader": "^0.6.2",
-        "awesome-typescript-loader": "^3.0.4-rc.0",
-        "babel-core": "^6.21.0",
-        "babel-preset-es2015": "^6.18.0",
-        "browserify": "^13.3.0",
-        "copy-webpack-plugin": "^4.0.1",
-        "css-loader": "^0.26.1",
-        "extract-text-webpack-plugin": "^2.0.0-rc.3",
-        "file-loader": "^0.10.0",
-        "glob": "^7.1.1",
-        "gulp": "^3.9.1",
-        "gulp-less": "^3.3.0",
-        "gulp-util": "^3.0.8",
-        "image-webpack-loader": "^3.2.0",
-        "less": "^2.7.2",
-        "less-loader": "^2.2.3",
-        "raw-loader": "^0.5.1",
-        "style-loader": "^0.13.1",
-        "tsify": "^3.0.0",
-        "typescript": "^2.1.5",
-        "vinyl-source-stream": "^1.1.0",
-        "webpack": "^2.2.1"
-    },
-    "dependencies": {
-        "@angular/common": "~2.4.0",
-        "@angular/compiler": "~2.4.0",
-        "@angular/core": "~2.4.0",
-        "@angular/platform-browser": "~2.4.0",
-        "@angular/platform-browser-dynamic": "~2.4.0",
-        "@angular/router": "^3.4.4",
-        "@angular/forms": "^2.4.4",
-        "@angular/http": "~2.4.3",
-        "@angular/material": "2.0.0-beta.1",
-        "@angular/flex-layout": "2.0.0-beta.4",
-        "core-js": "^2.4.1",
-        "es6-shim": "^0.35.2",
-        "reflect-metadata": "0.1.2",
-        "rxjs": "5.0.1",
-        "systemjs": "0.19.40",
-        "zone.js": "^0.7.4"
-    }
+  "name": "moninag",
+  "version": "1.0.0",
+  "description": "Monitoring system",
+  "main": "source_app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
we can use  jsonlint to have a proper format with 4 spaces 
`npm install --save-dev jsonlint` or smth like that, but it doesn't really matter.